test-page/explorer.js

```diff
@@ -54,6 +45,27 @@ function initExplorer(fileList) {
   );
 }
 
+var settings = {
+  labelColumn: 'filename',
+  ignoredColumns: ['local_path', 'rel_path', 'path'],
+  fileLoader: true,
+  metaFiles: ['AE', 'DM', 'LB'],
+  defaultCodebookSettings: {
+    autogroups: 2
+  }
+};
+
 document.onreadystatechange = function() {
-  initExplorer(dataFiles);
+
+  initExplorer(dataFiles, settings);
+  d3.select("body").append("p").text("Settings:")
+  d3.select("body").append("textarea")
+  .property("rows","10")
+  .property("cols",'100')
+  .property("value",JSON.stringify(settings))
+  .on("change",function(){
+    delete explorer;
+    d3.select("#container").selectAll("*").remove()
+    initExplorer(dataFiles, JSON.parse(this.value));
+  })
```

Comment:
@jwildfire  I don't see anything in the code yet, but I'm getting the error: "codebook.config.variableTypes is undefined" when I try to look at explorer.html. I'm guessing it's related to the meta type updates or it's my error?

Reply:
I'll take a look. #307